gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,7 +1,7 @@
-#Tue Jul 25 10:40:31 NOVT 2023
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-8.2.1-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.14.2-bin.zip
```

Comment:
I'm not sure if this is necessary for the project to build, but Android Studio was really unhappy about the old Gradle version.